src/readbin/byteweight_util.ml

```diff
@@ -13,7 +13,7 @@ let ignored = [
   ];
 ]
 
-module Byteweight = Bap_byteweight.Bytes
+module BW = Bap_byteweight.Bytes
```

Comment:
You mustn't use any `Bap_` in your code, this is internal modules. And actually, in upcomming BAP update, they're not available at all